Marlin/Configuration.h

```diff
@@ -2165,6 +2165,10 @@
 // Touch-screen LCD for Malyan M200/M300 printers
 //
 //#define MALYAN_LCD
+#if ENABLED(MALYAN_LCD)
+  // This will be 1 on a Malyan M200, but may differ on other boards.
+  #define MALYAN_LCD_SERIAL_PORT 1
+#endif
```

Comment:
Since we're not likely to ever have more than one serial LCD connected at the same time, we should probably combine all these "`JOHN_DOE_SERIAL_PORT`" options into a single "`LCD_SERIAL_PORT`" option.

Reply:
@ellensp and I were discussing this earlier since they have an Anycubic Chiron display you add as well. It’s certainly not sustainable to keep them separate if the number of displays keeps increasing.

I haven’t looked at your changes yet. The only real consideration between the different displays will be whether the emergency parser is enabled on them or not.

Reply:
I split out my serial cleanup changes into a separate commit and merged it, so this page will now only show the other changes.

As for emergency parser, it isn't a very costly feature and it only adds more logic to check whether it's enabled for a given serial port, so I don't see any issue leaving it enabled for serial ports with controllers on them.